octoprint_mrbeam/printing/comm_acc2.py

```diff
@@ -1603,6 +1588,39 @@ def _handle_user_command(self, cmd):
 			elif specialcmd.startswith('/correct_settings'):
 				self._log("Correcting GRBL settings...")
 				self.correct_grbl_settings()
+			elif specialcmd.startswith('/power_correction'):
```

Comment:
Is it possible to set another correction factor here?
It's not. Just double checking.